guides/integration-tool-release.md

```diff
@@ -30,4 +30,10 @@ Also, when writing the final changelogs, you might notice some inconsistencies,
 - Add the missing label (`breaking-change` or `skip-changelog`).
 - Re-run the last `Release Drafter` job in the `Actions` tab of the repository.
 
+> 💡 How to re-run the "Release Drafter" job? 
+> - Go to the `Action` tab.
+> - Click on the left sidebar `Release Drafter`.
+> - Select the latest action.
```

Comment:
Is it "always" the latest action? Or could this be more specific?

Reply:
It's always the latest action!

Reply:
I confirm